src/einsteinpy/geodesic.py

```diff
@@ -32,6 +32,12 @@ def __init__(
             return_cartesian=True,
         )[1]
 
+    def __repr__(self):
+        return {'body' : self.body, 'attractor' : self.attractor , 'metric': self.metric , '_trajectory': self._trajectory}
+
```

Comment:
`__repr__` should return a formal string representation of the object. I guess, you are returning a `dict`

Reply:
I guess teh trajectory must be skipped, it is an array. Just the details about the body, its parent body, its parent body's mass, and what metric it uses. That would work.

Reply:
I guess you also would like to add `self.body.parent.name` and `self.body.parent.mass`

Reply:
Yes on it , I assume this way of creating the `__str__` and `__repr__` methods is agreeable then?